crates/wadm-types/wit/deps/wadm/types.wit

```diff
@@ -123,6 +123,17 @@ interface types {
         id: option<string>,
         config: list<config-property>,
         secrets: list<secret-property>,
+        limits: list<limits-config>,
```

Comment:
We should bump the version of the package and add a `since` annotation on these. @brooksmtownsend any preference on whether we do a minor bump or a patch bump? I think with wit, if you add something to a `record` it is a breaking change, right?